motor_controller/setup.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+from setuptools import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+setup_args = generate_distutils_setup(
+    packages=['motor_controller'],
+    package_dir={'': 'src'},
+    install_requires=[
```

Comment:
Remove install_requires since it will be handled by the cmakelist dependencies instead

Reply:
Fixed.